### PR TITLE
Pedro review 1 of approval2template.pt.Rmd

### PR DESCRIPTION
**Pull request by pedropark99**
I made two changes considering the output of the translation API:

- Updated the paths to template documents that were referenced inside the `approval2template.pt.Rmd`;
- Manually added the translated version of the template document that was referenced inside the `approval2template.pt.Rmd`.

Following the translation policies of rOpenSci, we should do a one branch per file translated, so why did I added the translated version of the template document in this review? Because we need this template document to compile the book.

### CODE REVIEW

**Comment by yabellini** [comment]
Thank you Pedro for this work.  As you know, now we need a second review, so we will be looking for reviewers to finished the process before merge. :-)

**Review by msperlin — APPROVED** [review]

**Comment by pedropark99** [comment]
@maelle  as @msperlin  suggested in the comment below:

https://github.com/ropensci/dev_guide/pull/684#issuecomment-1774957274

So I started to check other files to see if we forgot to use a more "neutral gender" language. I found a point in the file of this PR where we needed to add the "(a)" suffix, so I added that in my last commit.

I understand @maelle  that this PR is now ready to be merged, since we did review 1 and 2 already, and we also did this last check that @msperlin  suggested.

File: templates/reviewer-approval.pt.md
Comment (yabellini) [inline templates/reviewer-approval.pt.md:2]: This should also be revisor(a) ?

File: templates/reviewer-approval.pt.md
Comment (yabellini) [inline templates/reviewer-approval.pt.md:6]: The same for autor?  (I'm sorry I don't know if autor is masculine only).

File: templates/reviewer-approval.pt.md
Comment (pedropark99) [reply templates/reviewer-approval.pt.md:2]: Good catch! Thank you!

File: templates/reviewer-approval.pt.md
Comment (pedropark99) [reply templates/reviewer-approval.pt.md:6]: Yes!

**Pull request merged**
